algorithm-simulator/src/main/scala/style95/scenarios/Demo.scala

```diff
@@ -1,9 +1,12 @@
 package style95.scenarios
+
 import style95.generator._
+import style95.scaler._
+
 import scala.concurrent.duration._
 
 object Demo extends ScenarioBase {
-  val secs = 10
-  val tps = 10
-  override def actorBuilder: BehaviorActorBuilder = Maintain(tps) in secs.seconds
+  override def actorBuilder: BehaviorActorBuilder = Maintain(10) in 10.seconds
+
+  override def scaler: Scaler = new FirstScaler(1000)
```

Comment:
What does first mean here?

Reply:
It's the *first* scaler that has been proposed :smile. I'm bad at naming.

Reply:
Got it.
How about `DefaultScaler`?

Reply:
Great!